app/src/main/java/com/github/dedis/student20_pop/home/HomeViewModel.java

```diff
@@ -193,4 +242,16 @@ public void openConnect() {
     public void openLaunch() {
         mOpenLaunchEvent.setValue(new Event<>(true));
     }
+
+    public void launchNewLao() {
+        mLaunchNewLaoEvent.setValue(new Event<>(true));
+    }
+
+    public void cancelNewLao() {
+        mCancelNewLaoEvent.setValue(new Event<>(true));
+    }
+
+    public void setLaoName(Editable name) {
```

Comment:
```suggestion
    public void setLaoName(String name) {
```

Would make unit testing easier :)